docs/source/1.0/spec/core/shapes.rst

```diff
@@ -1294,14 +1303,14 @@ structure named ``Output``:
 Operation errors
 ----------------
 
-The errors of an operation is an optional array of shape IDs that MUST
-target structure shapes that are marked with the :ref:`error-trait`. Errors
-defined on an operation are errors that can potentially occur when calling
-an operation.
+The ``errors`` of an operation is an optional set of shape IDs that MUST
```

Comment:
Should this be "errors property of an operation" or "an operation are" to handle plurality?